docs/helpers/env_vars.yaml

```diff
@@ -1,3 +1,149 @@
+ACTIVITYLOG_CORS_ALLOW_CREDENTIALS:
+  name: OCIS_CORS_ALLOW_CREDENTIALS;ACTIVITYLOG_CORS_ALLOW_CREDENTIALS
+  defaultValue: "true"
+  type: bool
+  description: 'Allow credentials for CORS.See following chapter for more details:
+    *Access-Control-Allow-Credentials* at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials.'
+  introductionVersion: pre5.0
```

Comment:
Maybe I do not understand the process.
How can these new lines have `introductionVersion: pre5.0` ?

Reply:
The process is the following:

`make -C docs docs-generate` parses all the envvars from all services and adds them to a kind of "database" file (this one) for further actions we need to generate the final .adoc files. The helper takes whatever devs have done with envvars in services, therefore the helper is "neutral" about the content. To distinguish when an envvar has been added, we needed to decide which one have been introduced before 5.0. Having that, we can now clearly identify when an envvar has been introduced, respctively will get deprecated.

With an upcoming version of the helper, we will add to the adoc output WHEN the respective envvar got introduced in the admin docs. Also, we can now provide easily a list of introduced envvars for a particular release 😄 

Hope this makes sense.